src/style.css

```diff
@@ -0,0 +1,32 @@
+li {
+  height: 10vh;
+}
+
+ul {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  text-align: right;
+  width: 10em;
+}
+
+input {
+  position: relative;
+}
+
+.task-placeholder {
+  align-items: center;
+  justify-content: space-between;
+}
+
+.input-task {
+  border-top-color: black;
+  border-top-width: 1px;
+  border-top-style: solid;
+}
+
+.tasks-itm {
+  list-style-type: none;
+  display: flex;
+}
```

Comment:
One of the requirements states that CSS is created used to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Please note that the video has more features than the current project, students don't need to style all the features not implemented yet in the video. CSS must be loaded by *Webpack Style/CSS Loader*. here 👇 
![image](https://user-images.githubusercontent.com/65068771/150223070-ed2c160a-02ac-4bb5-add1-9d4b57ad97ae.png)

- [ ] Kindly you need to follow the styling in the video demonstration as the requirement says. here is your design 👇 
![image](https://user-images.githubusercontent.com/65068771/150222609-27354450-8d8d-40b2-8da7-1ce53dbb6cdc.png)